### ISSUE TITLE
Add card "found in sealed product" section.

### ISSUE THREAD

**Issue by axxroytovu** [taw/magic-search-engine#269]
Now that MTGJson hosts a "source products" object, it would be great to have that displayed on the card page in some way. 

E.g. from Archon of the Wild Rose:
```
"sourceProducts": {"foil": ["1ff62823-f41c-5334-af14-20c5cc28e84d", "86ab9007-922a-5f13-8ffa-aaa8b775c06c"], "nonfoil": ["1ff62823-f41c-5334-af14-20c5cc28e84d", "86ab9007-922a-5f13-8ffa-aaa8b775c06c"]}
```

**Comment by taw** [comment]
Yeah, I'd like to integrate the whole system someday and [not just leave it like this](https://mtg.wtf/product), but I don't have time to do the development right now.

**Comment by taw** [comment]
I sort of have products now, so I could possibly progress this.

**Comment by taw** [comment]
This feature is live now.

**Comment by taw** [comment]
[You can even check all printings together](https://mtg.wtf/card/availability/prm/70934)
